### ISSUE TITLE
Data Processing and Pytorch information 

### ISSUE THREAD

**Issue by Nitin-Mane** [AMLResearchProject/ALL-PyTorch-Segmentation-2021#14]
**Is your feature proposal related to a problem? Please describe.**

Data processing is the necessary part for the deep learning procedure and while the data is limited therefore some alternative operations choose to execute.

**Describe the solution you'd like to implement**
A clear and concise description of what you want to make happen.

The data analysis is based on the first update of the implementation required for the data processing tools. As using Pytorch therefore it can be loaded in batch and split into training, test and validation or else create a subfolder under the data and spilt data accordingly. This procedure requires augmentation via masking images that have the annotation features or the pixel selection mapping techniques. 

**Describe alternatives you've considered**

The alternative for the data processing is to consider the batch size process and add a features in the scripts for the development of the scripting files and libraries for the process

**Additional context**

The data processing require a certain space and file configuration will be updated accordinly to the procedure.
